build/template_kql/common/summarize_score.kql

```diff
@@ -30,11 +30,13 @@ summarize
     UnknownBackupCount=dcountif(Name, (Backup contains "Unknown")),
     HealthyBackupCount=dcountif(Name, (Backup == "healthy")),
     UnhealthyBackupCount=dcountif(Name, (Backup == "unhealthy")),
+    EnabledBackupCount=dcountif(Name, (Backup == "enabled")),
+    DisabledBackupCount=dcountif(Name, (Backup == "disabled")),
     RunningState=dcountif(Name, State contains "running" or State contains "ready"),
     DBOnlineStateCount=dcountif(Name, State contains "online" or State contains "ready"), // online - SQL, Ready - MySQL
     DBPausedStateCount=dcountif(Name, State contains "paused"),
-    DBGeoStorage=dcountif(Name, OSDisk == "grs"),
-    DBLrsStorage=dcountif(Name, OSDisk == "lrs"),
+    DBGeoStorage=dcountif(Name, OSDisk == "geo"),
+    DBLrsStorage=dcountif(Name, OSDisk != "geo"),
```

Comment:
DBLrsStorage=dcountif(Name, OSDisk != "geo"),

This line counts non-geo which might include ZRS storage as well. Is it intended for LRS count?

Reply:
I checked the variable again, DBGeoStorage is currently used to calculate the SQL DB score and DBLrsStorage is not used anywhere.
```
    // SQLDB - 4 checks  // score = ("status" + "tier" + "storage acct. type"  + "AZ")* 100 / (5 * ResourceTotal)  //********* Future - "read replica count"
    Type == 'microsoft.sql/servers/databases', (DBOnlineStateCount + SqlPremiumOrBusinessCriticalOrDwh + DBGeoStorage + AvZoneCount + NAAvZoneCount) * 100 / (4 * ResourceTotal),
```
So we can delete DBLrsStorage, but I'd like to keep it just in case.